scripts/build_debs.sh

```diff
@@ -8,16 +8,24 @@
 set -e
 
 # Manually pinned to known good version, see issue #17 for history
-CARGO_DEB_VERSION="2.9.4"
+CARGO_DEB_VERSION="2.11.2"
 
-# The target triplet(s) must be given as args.
+# The target triplet must be given as an arg, for example: x86_64-unknown-linux-gnu
+target="${1?}"
```

Comment:
Is there a handler for "you didn't specify a target!"? Well, other than "it doesn't do anything..."

Reply:
The question mark makes it barf as param missing, but might as well give it a helpful error as we may have humans doing it.